src/components/EventMap.js

```diff
@@ -84,6 +84,8 @@ export default function(store){
     },
     methods: {
       plotEvents() {
```

Comment:
A couple potential race conditions guarded against -- if these are called but the map hasn't yet initialized, return.